keyboards/westfoxtrot/prophet/prophet.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include "quantum.h"
+
+#define ___ KC_NO
+
+#define LAYOUT( \
```

Comment:
As the PCB supports multiple physical layouts, this should be named `LAYOUT_all`.

```suggestion
#define LAYOUT_all( \
```